app/assets/stylesheets/petitions/views/_shared.scss

```diff
@@ -230,17 +230,17 @@ input.back-page {
 .debate-video {
   position: relative;
   width: 100%;
-  padding-bottom: 56.25%; // 16:9
-  background-size: 100%;
   @include inline-block;
-  background-image: image-url("graphics/graphic_house-of-commons.jpg");
-  @include device-pixel-ratio() {
-    background-image: image-url("graphics/graphic_house-of-commons-2x.jpg");
-  }
   &:hover, &:focus {
     opacity: 0.9;
   }
 
+  img {
+    width: 100%;
+    height: 100%;
+    vertical-align: bottom;
+  }
+
```

Comment:
The way I'd normally do the play button would be to `position: relative` the wrapper `<div>` and then `position: absolute` the play button with `left: 0; bottom: 0`. But if this works and Henry's okayed it then there's no need to change it.